crates/topos-metrics/src/lib.rs

```diff
@@ -31,3 +31,15 @@ lazy_static! {
     pub static ref CERTIFICATE_DELIVERED: IntCounter =
         register_int_counter!("certificate_delivered", "Number of certificate delivered.").unwrap();
 }
+
+pub async fn gather_metrics() -> String {
```

Comment:
I don't see why this is `async`, is it to directly use it as a `handler` in `axum` to avoid wrapping a sync method in an `async { .. }` ?

Reply:
Yes! But I made it `sync` now and moved it in an `async` block 👍  Much more obvious this way, thanks!